src/PointwiseFunctions/AnalyticSolutions/ForceFree/ExactWald.hpp

```diff
@@ -0,0 +1,175 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <memory>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/ForceFree/Tags.hpp"
+#include "Options/Context.hpp"
+#include "Options/String.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/AnalyticSolution.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/KerrSchild.hpp"
+#include "PointwiseFunctions/GeneralRelativity/KerrSchildCoords.hpp"
+#include "PointwiseFunctions/InitialDataUtilities/InitialData.hpp"
+#include "Utilities/Serialization/CharmPupable.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+class DataVector;
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace ForceFree::Solutions {
+
+/*!
+ * \brief An exact electrovacuum solution of Maxwell's equations in the Kerr
+ * spacetime found by Wald \cite Wald1974.
```

Comment:
Kerr -> Schwarzschild?  The paper is valid for arbitrary spin, but only the zero-spin case is implemented (and I take it that the spinning case isn't a solution to the force-free equations.)

Reply:
Updated the docs